spec/models/form526_submission_spec.rb

```diff
@@ -1282,10 +1282,24 @@ def expect_no_max_cfi_logged(diagnostic_code)
       context 'with submission confirmation email when successful job statuses' do
         subject { create(:form526_submission, :with_multiple_succesful_jobs) }
 
-        it 'returns one job triggered' do
-          expect do
-            subject.workflow_complete_handler(nil, 'submission_id' => subject.id)
-          end.to change(Form526ConfirmationEmailJob.jobs, :size).by(1)
+        context 'with disability_526_call_received_email_from_polling enabled' do
+          it 'does not trigger confirmation email job' do
+            expect do
+              subject.workflow_complete_handler(nil, 'submission_id' => subject.id)
+            end.to change(Form526ConfirmationEmailJob.jobs, :size).by(0)
+          end
+        end
+
+        context 'with disability_526_call_received_email_from_polling disabled' do
+          before do
+            Flipper.disable(:disability_526_call_received_email_from_polling)
```

Comment:
Can you mock this instead? See [this example](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample) in the spec. 
```suggestion
            allow(Flipper).to receive(:enabled?).with(:disability_526_call_received_email_from_polling).and_return(false)
```
enabling and disabling in tests can make other tests fail since we run the tests in parallel in CI.

Reply:
Made the change!